### PR TITLE
Plant miappe

### PR DESCRIPTION
**Pull request by FuqiX**
Hi, @proccaserra 
This is the plant data submission recipe by @ErwanLeFloch  @sebeier 
Please have a look.

### CODE REVIEW

File: content/recipes/reusability/miappe.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+## Graphical overview of the Recipe FAIRification Objectives
+![](https://i.imgur.com/iqdjWqo.png)
+
+
+
+## Capability & Maturity Table
+|Capability|Initial Maturity Level|Final Maturity Level|
+|--- |--- |--- |
+|Interoperability|minimal|repeatable|
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:34]:
```suggestion
## Capability & Maturity Table
|Capability|Initial Maturity Level|Final Maturity Level|
|--- |--- |--- |
|Interoperability|minimal|repeatable|
```

TODO: check with @iemam and perform a mapping against the dataset maturity indicators https://github.com/FAIRplus/Data-Maturity/blob/master/docs/Indicators.md

to clearly identify which are the maximum level of maturity reached if one applies to recipe.
[F+MM-2.3R] : machine readable metadata
[F+MM-3.3C]: dataset is given of global identifier 
could be reached but I don't think there is evidence in the recipe of the following:
[F+MM-4.1R]: Metadata is represented in a semantic machine interpretable form. (is there a jsonld for the same information?)

File: content/recipes/reusability/miappe.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+[TOC]
+
+
+## Main Objective
+
+Plant genotyping and phenotyping data are often used in combination to make evidence-based inferences about different trait expressions. A challenge here is not necessarily to collect the data, but to offer them in a stable way over the long term in public repositories with sufficient metadata information to make conclusions about them in line with FAIR principles. Since these repositories are often not directly linked, it is even more important to provide metadata that allows users to recognise these links by their identifiers. A key point here is sample management; the identifiers assigned here help both humans and machines to understand which experimental data are linked.
+
+The main objective of the recipe is to provide a means of submitting and tracking genotyping data in public repositories, with a particular focus on plants. This includes:
+
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:11]:
```suggestion
The main objective of the recipe is to provide a means of submitting to public repositories and tracking genotyping data, with a particular focus on plants.  This includes:

```

File: content/recipes/reusability/miappe.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+    1) Submission of sample data and metadata information to BioSamples.
+    2) Submission of sequencing data and metadata to ENA.
+    3) Retrieval of the correct genome assembly for the genotyping experiment
+    4) Conversion of the resulting analysis file (in VCF format) to be FAIR
+    5) Submission of the genotyping results to EVA.
+
+
+
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:19]:
```suggestion

In terms of FAIRification goals, this means obtaining stable, resolvable  identifiers for the datasets and meeting community annotation requirements as expressed in the MIAPPE requirements. 
```

File: content/recipes/reusability/miappe.md [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+## Summary
+
+This recipe provides guidance for submitting plant genotyping data to public repositories. It explains step-by-step which work should be done and when. Special attention should be paid to the metadata maintenance of the data that will be deposited in different repositories as part of this recipe. A prerequisite for fully understanding this recipe is a basic knowledge of the [MIAPPE standard](https://fairsharing.org/bsg-s000543/).
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:22]:
```suggestion
This recipe provides guidance for submitting plant genotyping data to public repositories. It explains in a step-wise fashion which work should be done and when. Special attention should be paid to the metadata maintenance of the data that will be deposited in different repositories as part of this recipe. A prerequisite for fully understanding this recipe is a basic knowledge of the [MIAPPE standard](https://fairsharing.org/bsg-s000543/). 
```

File: content/recipes/reusability/miappe.md [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+
+
+#### Step 1.2: Register sample material at BioSamples
+
+This is done in general through JSON API ([Python](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/python) and [Shell](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/shell) commands are also available). Refer to the official [documentation](https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_a_sample) for the complete details. Here is the proposed procedure:
+
+1. Create a user account
+
+First you need an account to submit samples to BioSamples. We recommend new users, or those planning to make downstream submissions to ENA, use the Webin Authentication service. You can create a Webin account using [Webin web interface](https://www.ebi.ac.uk/ena/submit/sra/#home). Please refer to [ENA documentation](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/registration.html) for more details about creating an account.
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:86]:
```suggestion
First you need an account to submit samples to EMBL-EBI BioSamples database. We recommend new users, or those planning to make downstream submissions to ENA, to use the Webin Authentication service. You can create a Webin account using [Webin web interface](https://www.ebi.ac.uk/ena/submit/sra/#home). Please refer to [ENA documentation](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/registration.html) for more details about creating an account. 
```

File: content/recipes/reusability/miappe.md [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Step 1.2: Register sample material at BioSamples
+
+This is done in general through JSON API ([Python](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/python) and [Shell](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/shell) commands are also available). Refer to the official [documentation](https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_a_sample) for the complete details. Here is the proposed procedure:
+
+1. Create a user account
+
+First you need an account to submit samples to BioSamples. We recommend new users, or those planning to make downstream submissions to ENA, use the Webin Authentication service. You can create a Webin account using [Webin web interface](https://www.ebi.ac.uk/ena/submit/sra/#home). Please refer to [ENA documentation](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/registration.html) for more details about creating an account. 
+
+2. Login to the system and get a JWT
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:88]:
```suggestion
2. Login to the system and get a JSON Web Token (JWT)
```

File: content/recipes/reusability/miappe.md [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+
+## Table of Software and Tools
+|Tool Name|
+|--- |
+|[BioSamples](https://www.ebi.ac.uk/biosamples/)|
+|[European Nucleotide Archive, ENA](https://www.ebi.ac.uk/ena/browser/home)|
+|[European Variation Archive,EVA](https://www.ebi.ac.uk/eva/)|
+|[International Nucleotide Sequence Database Collaboration](https://www.insdc.org/)
+|[VCF Validator](https://github.com/EBIvariation/vcf-validator/wiki/User's-Guide)|
+
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:67]:
```suggestion
|[Curl](https://curl.se/)|
```

File: content/recipes/reusability/miappe.md [hunk 8]
```diff
@@ -0,0 +1,429 @@
...
+6. Now navigate to the '**Studies**' tab to see the study you just registered. You might need to refresh the page!  
+Make a note of its accession numbers (_ERP#####_, _PRJEB#####_), which will resemble. For a submission, these would be the numbers you would cite in any publications involving the data.
+
+##### The Sample
+
+The next step is to register the sample, which will give other users essential context for the sequence data you are submitting. The sample describes the source biological material of your sequencing work.
+
+As discussed above, **samples are best submitted through BioSamples**.
+
+In ENA, samples are required to conform to a checklist of values. Checklists define a set of mandatory and recommended values for a given type of sample. It is recommended that you look at these early and make sure you collect all required metadata items for the type of sample you will be registering.
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:149]:
```suggestion
In ENA, samples are required to conform to a checklist of values. Checklists define a set of mandatory and recommended descriptor fields for a given type of sample. It is recommended that you look at these early and make sure you collect all required metadata items for the type of sample you will be registering.
```

File: content/recipes/reusability/miappe.md [hunk 9]
```diff
@@ -0,0 +1,429 @@
...
+        Barley sample from <Your Town/Lab> greenhouse
+
+10. Write a brief description for your sample.
+11. Find the appropriate taxon for your sample: For example enter '**Hordeum vulgare**' into the box and select it when it appears. 
+
+    Further taxonomic details are automatically filled below this box. This box will only accept values which match with a species-rank entry in the [NCBI Taxonomy database](https://www.ncbi.nlm.nih.gov/taxonomy).
+
+12. Fill out the remaining details by reference to the values to the best of your knowledge. Only fields marked with a ***** are required, but it is best to fill as many fields as possible.
+
+    **Hint:** You can always find out more about what a field means and what information it accepts by hovering over the blue 'i'.
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:189]:
````suggestion
Use a markdown directive (note: not supported by hackMD but supported by jupyter notebook)
``` {note}
You can always find out more about what a field means and what information it accepts by hovering over the blue 'i'.
```
````

File: content/recipes/reusability/miappe.md [hunk 10]
```diff
@@ -0,0 +1,429 @@
...
+Submit Sequencing reads to ENA, using BioSamples IDs to identify material.
+
+
+##### The Study
+
+To begin, you should register a study. Recall that a study describes the purpose of the work you have done, groups other objects beneath it, and controls when the data becomes public. A study is required for all submissions to ENA.
+
+1. Log in to the [Webin Submission Portal](https://www.ebi.ac.uk/ena/submit/sra/#home) with your Webin credentials.
+ 
+2. Click the '**New Submissio**n' tab and find the '**Register study (project)**' radio button. Click '**Next**' to see the study registration interface:
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:131]:
```suggestion
2. Click the '**New Submission**' tab and find the '**Register study (project)**' radio button. Click '**Next**' to see the study registration interface:
```

File: content/recipes/reusability/miappe.md [hunk 11]
```diff
@@ -0,0 +1,429 @@
...
+    * **Instrument Model:** Illumina HiSeq 2500
+    * **Library Name:** barley_library_1
+    * **Library Source:** GENOMIC
+    * **Library Selection:** Restriction Digest
+    * **Library Strategy:** GBS
+    * **Library Layout:** SINGLE
+    * **First File Name:** HOR_1361_BRG.fastq.gz
+
+
+    **Hint**: If you did not note down your sample accession, you can safely visit the ‘**Samples**’ tab, copy the accession, then return to the New Submission tab: your progress will be saved.
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:231]:
````suggestion
```{note}
 If you did not note down your sample accession, you can safely visit the ‘**Samples**’ tab, copy the accession, then return to the New Submission tab: your progress will be saved.
```
````

File: content/recipes/reusability/miappe.md [hunk 12]
```diff
@@ -0,0 +1,429 @@
...
+
+7. Click '**Submit**' and see if your submission validates successfully. If you encounter errors, try using the '**Download Template Spreadsheet'** button, open the file and check it in this plain text format; it can be easier to fix errors this way than in the interface. 
+
+
+    Note that the use of template spreadsheets is the best way to submit multiple datasets through this interface.
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:236]:
````suggestion
```{note}
Note that the use of template spreadsheets is the best way to submit multiple datasets through this interface.
```
````

File: content/recipes/reusability/miappe.md [hunk 13]
```diff
@@ -0,0 +1,429 @@
...
+
+
+##### Recommended meta-information lines :
+
+**##fileDate(Date)**: creation date of the VCF in the basic form without separator: YYYYMMDD \
+\
+Example: \
+  `##fileDate=20120921` 
+
+**##bioinformatics_source (URL or URI)**: Analytic approach usually consisting of chains of bioinformatics tools for creating the VCF file specified as the DOI of a publication, or more generally as URL/URI, like a public repository for the scripts used. \
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:280]: suggestion: state something about depositing Workflows to Workflowhub.eu , possibly in CWL format.

In so doing, we'll be able to link to the relevant Recipe about Workflows in the FAIR Cookbook . Tx

File: content/recipes/reusability/miappe.md [hunk 14]
```diff
@@ -0,0 +1,429 @@
...
+
+
+##### In case of adding new fields : 
+
+Please check the official format specifications to avoid redundancy and possible incompatibilities.
+
+
+#### Step 4.3: Submit VCF file to EVA
+
+Once the metadata and data has been formatted according to the specifications above, make sure that the resulting VCF file complies to VCF specifications. For that purpose we propose the [VCF validator](https://github.com/EBIvariation/vcf-validator) (an example on how to use it is available on the github link). \
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:332]:
```suggestion
Once the metadata and data has been formatted according to the specifications above, make sure that the resulting VCF file complies with VCF specifications. For that purpose, we propose the [VCF validator](https://github.com/EBIvariation/vcf-validator) (an example of how to use it is available on the github link). \
```

File: content/recipes/reusability/miappe.md [hunk 15]
```diff
@@ -0,0 +1,429 @@
...
+    \
+Once the file has been fully validated without any error messages you can submit the VCF file to EVA, using BioSamples IDs to identify material, GCF/GCA accession for the reference genome assembly, and ENA accession numbers for the sequencing reads of the material used. Refer to the official [documentation](https://www.ebi.ac.uk/eva/?Submit-Data).
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:334]:
```suggestion
Once the file has been fully validated without any error messages you can submit the VCF file to EVA, using BioSamples IDs to identify the material, GCF/GCA accession for the reference genome assembly, and ENA accession numbers for the sequencing reads of the material used. Refer to the official [documentation](https://www.ebi.ac.uk/eva/?Submit-Data).
```

File: content/recipes/reusability/miappe.md [hunk 16]
```diff
@@ -0,0 +1,429 @@
...
+  </tr>
+</table>
+
+
+
+
+## Licence
+
+[https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/)
+
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:429]:
```suggestion

````{license_fairplus}
CC-BY-4.0
````
[https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/)

```

File: content/recipes/reusability/miappe.md [hunk 17]
```diff
@@ -0,0 +1,429 @@
...
+[Beier et. al (2022) - Recommendations for the formatting of VCF files to make plant genotyping data FAIR](https://doi.org/10.12688/f1000research.109080.1)
+
+[RDMkit - Tools Assembly - Plant Genomics](https://rdmkit.elixir-europe.org/plant_genomics_assembly.html)
+
+[Training Material from FONDUE datathon 2021](https://github.com/PBR/elixir-fondue-datathon)
+
+
+
+## Authors
+
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:350]: TODO: @proccaserra to replace with correct directive

**Review by proccaserra — CHANGES_REQUESTED** [review]
@FuqiX @ErwanLeFloch @sebeier thank you for your contribution. Only a couple of minor changes and things the FAIR Cookbook editor will need to liaise with you in order to complete.

**Review by ErwanLeFloch — APPROVED** [review]
Hi @proccaserra , 
thank you for your feedback and sorry for my late answer.
Cheers,
Erwan

File: content/recipes/reusability/miappe.md (hunk 1 above)
Comment (ErwanLeFloch) [reply content/recipes/reusability/miappe.md:33]: The levels [F+MM-2.3R] and [F+MM-3.3C] are reached, but [F+MM-4.1R] is not. 
There is a json schema: https://www.ebi.ac.uk/biosamples/schemas/certification/plant-miappe.json  , but no jsonld.

File: content/recipes/reusability/miappe.md [hunk 18]
```diff
@@ -0,0 +1,435 @@
...
+The following CURL command returns 3 accessions as the body contains names for 3 samples.
+
+`curl 'https://www.ebi.ac.uk/biosamples/samples/bulk-accession?authProvider=WEBIN' -i -X POST -H "Content-Type: application/json;charset=UTF-8" -H "Accept: application/hal+json" -H "Authorization: Bearer $TOKEN" -d '[{  "name" : "FakeSample 1"}, { "name" : "FakeSample 2"}, {  "name" : "FakeSample 3"}]'`
+
+Please refer to the [BioSamples documentation](https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_a_sample). 
+
+
+
+More general information is available on the RDMkit [plant genomics assembly](https://rdmkit.elixir-europe.org/plant_genomics_assembly.html). A specific checklist is used: [BioSamples - Plant MIAPPE checklist](https://www.ebi.ac.uk/biosamples/schemas/certification/plant-miappe.json).
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:108]:
```suggestion
More general information is available on the RDMkit {footcite}`rdmkit_plant_genomics_assembly`. A specific checklist is used: [BioSamples - Plant MIAPPE checklist](https://www.ebi.ac.uk/biosamples/schemas/certification/plant-miappe.json).
```

File: content/recipes/reusability/miappe.md [hunk 19]
```diff
@@ -0,0 +1,435 @@
...
+
+
+#### Step 4.1: Analyse sequencing results
+
+The bioinformatician performs the computational analysis, then the genotyping results are archived into the LIMS.
+
+
+#### Step 4.2: Prepare genotyping dataset for submission
+
+Refer to the publication [Recommendations for the formatting of VCF files to make plant genotyping data FAIR](https://doi.org/10.12688/f1000research.109080.1).
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:266]:
```suggestion
Refer to the publication {footcite}`beier_f1000r_2022`.
```

File: content/recipes/reusability/miappe.md [hunk 20]
```diff
@@ -0,0 +1,435 @@
...
+
+
+#### Step 1.1: Take DNA sample
+
+The experimentalist takes a sample of plant biological material. The sample metadata are collected according to _the [MIAPPE](https://github.com/MIAPPE/MIAPPE/tree/master/MIAPPE_Checklist-Data-Model-v1.1) specifications, Biological Material section_. It enriches minimal [MCPD](https://www.bioversityinternational.org/e-library/publications/detail/faobioversity-multi-crop-passport-descriptors-v21-mcpd-v21/) fields with sample traceability information.
+
+
+#### Step 1.2: Register sample material at BioSamples
+
+This is done in general through JSON API ([Python](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/python) and [Shell](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/shell) commands are also available). Refer to the official [documentation](https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_a_sample) for the complete details. Here is the proposed procedure:
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:81]:
```suggestion
This is done in general through JSON API ([Python](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/python) and [Shell](https://github.com/PBR/elixir-fondue-datathon/tree/master/test_data_set/BioSamples/scripts/shell) commands are also available). Refer to the official [documentation](https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_a_sample) for the complete details, for training material and slides regarding this refer to {footcite}`fondue_datathon`. Here is the proposed procedure:
```

File: content/recipes/reusability/miappe.md [hunk 21]
```diff
@@ -0,0 +1,435 @@
...
+   </td>
+   <td><a href="https://orcid.org/0000-0002-2177-8781">0000-0002-2177-8781</a>
+   </td>
+   <td>Writing - Original Draft
+   </td>
+  </tr>
+  <tr>
+   <td>Erwan Le Floch
+   </td>
+   <td>URGI (Unité de Recherche Génomique Info), INRAE,
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:384]:
```suggestion
   <td>Université Paris-Saclay, INRAE, BioinfOmics, Plant bioinformatics facility, 78026, Versailles, France
```

File: content/recipes/reusability/miappe.md [hunk 22]
```diff
@@ -0,0 +1,435 @@
...
+Route de Saint-Cyr, 78026 Versailles, France
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:385]:
```suggestion

```

File: content/recipes/reusability/miappe.md [hunk 23]
```diff
@@ -0,0 +1,435 @@
...
+   </td>
+   <td><a href="https://orcid.org/0000-0002-1010-6859">0000-0002-1010-6859</a> 
+   </td>
+   <td>Writing
+   </td>
+  </tr>
+  <tr>
+   <td>Cyril Pommier
+   </td>
+   <td>URGI (Unité de Recherche Génomique Info), INRAE,
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:395]:
```suggestion
   <td>Université Paris-Saclay, INRAE, BioinfOmics, Plant bioinformatics facility, 78026, Versailles, France
```

File: content/recipes/reusability/miappe.md [hunk 24]
```diff
@@ -0,0 +1,435 @@
...
+Route de Saint-Cyr, 78026 Versailles, France
```
Comment (ErwanLeFloch) [inline content/recipes/reusability/miappe.md:396]:
```suggestion

```

**Review by ErwanLeFloch — APPROVED** [review]

**Review by ErwanLeFloch — APPROVED** [review]
@proccaserra we edited the bibliography with @sebeier

**Review by ErwanLeFloch — CHANGES_REQUESTED** [review]
@proccaserra could you confirm our suggestions ?

**Review by ErwanLeFloch — APPROVED** [review]

**Review by ErwanLeFloch — APPROVED** [review]

**Review by ErwanLeFloch — APPROVED** [review]

**Comment by ErwanLeFloch** [comment]
@FuqiX @proccaserra  can the pull request be merged ?

**Comment by proccaserra** [comment]
@ErwanLeFloch @FuqiX  I have merged your contribution into another branch (which contains a major upgrade).
So we could actually close this PR.
I have made further minor tweaks (changing headers slightly) to match the style  with the rest of the book.
I will issue the identifier and notify you when done with all this. Thank you again.

**Comment by ErwanLeFloch** [comment]
@proccaserra Great ! Thanks :+1:

**Pull request closed without merging**